File: android/employer/app/src/main/java/com/workforcex/employer/ui/auth/SplashActivity.java
package com.workforcex.employer.ui.auth;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.workforcex.employer.ui.employer.EmployerDashboardActivity;
import com.workforcex.employer.utils.TokenManager;

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TokenManager tokenManager = new TokenManager(this);
        if (tokenManager.isLoggedIn()) {
            startActivity(new Intent(this, EmployerDashboardActivity.class));
        } else {
            startActivity(new Intent(this, LoginActivity.class));
        }
        finish();
    }
}

File: android/employer/app/src/main/java/com/workforcex/employer/ui/auth/SplashActivity.kt
package com.workforcex.employer.ui.auth

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.workforcex.employer.ui.employer.EmployerDashboardActivity
import com.workforcex.employer.utils.TokenManager

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val tokenManager = TokenManager(this)
        if (tokenManager.isLoggedIn) {
            startActivity(Intent(this, EmployerDashboardActivity::class.java))
        } else {
            startActivity(Intent(this, LoginActivity::class.java))
        }
        finish()
    }
}